src/app/@ansyn/map-facade/actions/map.actions.ts

```diff
@@ -420,3 +503,13 @@ export class SetOverlaysFootprintActive implements Action {
 	constructor(public payload: {mapId: string, show: boolean}) {}
 
 }
+
+export class LogMessageFromImagery implements Action, ILogMessage {
```

Comment:
can you explain this log?

Reply:
Sure. This log is for sending log message from "imagery" that is in fact from our packages that do not use store (ngrx). So I cannot send actions directly from code in those packages. Instead I am using the "communicator" objects. See below in file `annotation-context-menu.component.ts`.  In the code I am sending a log message via an event emitter under the communicator. In service `map-facade.service.ts` I am subscribing to this event emitter (and other event emitters of the communicator). When I get a log message from the emitter, I am sending this action LogMessageFromImagery, and by this the log message goes to the logger.